invokeai/app/api/routers/model_manager.py

```diff
@@ -848,74 +846,6 @@ async def get_starter_models() -> StarterModelResponse:
     return StarterModelResponse(starter_models=starter_models, starter_bundles=starter_bundles)
 
 
-@model_manager_router.get(
-    "/model_cache",
-    operation_id="get_cache_size",
-    response_model=float,
-    summary="Get maximum size of model manager RAM or VRAM cache.",
-)
-async def get_cache_size(cache_type: CacheType = Query(description="The cache type", default=CacheType.RAM)) -> float:
```

Comment:
@psychedelicious AFAICT, the `get_cache_size(...)` and `set_cache_size(...)` endpoints weren't being called anywhere in the frontend. Is that accurate? With the changes to how the cache size is managed, it was easiest to just rip these out

Reply:
Yep that's correct.